pages/index.vue

```diff
@@ -9,12 +9,18 @@
         <b-col cols="12" :lg="panelSize" :offset-lg="panelOffset" style="transition: all 500ms">
           <div class="p-3">
             <b-form-file
-              v-model="splitFile"
+              v-model="fileInput"
               accept=".lss"
               placeholder="Choose a file or drop it here..."
               drop-placeholder="Drop file here..."
-              class="mb-4"
+              class="mb-3"
             ></b-form-file>
+            <p>or import splits from splits.io:</p>
```

Comment:
It would be better to make the whole thing an actual [form](https://bootstrap-vue.org/docs/components/form/) (potentially [inline](https://bootstrap-vue.org/docs/components/form/#inline-form) to keep the same visual aspect) and use the `@submit` event to load stuff instead of the `@click` event of the button, that way people can either click the button or press enter and it'll work the same

Also it's more of a subjective thing but I think it would look better if everything was on one line (including the button, kind of similar to the "rename comparison" button) and if the text input was a lot smaller since splits.io IDs are only 4 characters long